vectorization/simple_vectorization.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+
+def simple_vectorization(spectrum):
```

Comment:
The same note on simple_vectorization.py and simple_vectorization2.py - those do not need to be separated, you can unite them into one class.